int64 num_of_gestures
---
int64[] gesture_sequence